### PR TITLE
Adding Stale bot to GitHub repository

### PR DESCRIPTION
**Pull request by pauarge**
## Description

This PR adds the Stale bot to this repository. We are using the default configuration, which means that PRs will be marked as stale after 60 days of inactivity and will be closed 7 days after.

Following the [instructions of the bot](https://github.com/apps/stale), I have already enabled the app on the repository.

Note that this PR only affects the GitHub repository thus it doesn't need to be deployed.

### Adding Stale bot to GitHub repository

Internal tooling update to better track external contributions.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ ] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [x] I've created a changelog description that aligns with the provided examples.

### CODE REVIEW

File: .github/.stale [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
```
Comment (rinatkhaziev) [inline .github/.stale:2]:
```suggestion
daysUntilStale: 90
```

File: .github/.stale [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```
Comment (rinatkhaziev) [inline .github/.stale:4]:
```suggestion
daysUntilClose: 14
```

File: .github/.stale [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# Label to use when marking an issue as stale
+staleLabel: [Status] Stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity in the past 60 days. It will be closed in 7 days if no
```
Comment (rinatkhaziev) [inline .github/.stale:10]:
```suggestion
  recent activity in the past 90 days. It will be closed in 14 days if no
```

File: .github/.stale [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  further activity occurs. Thank you for your contributions.
```
Comment (rinatkhaziev) [inline .github/.stale:11]:
```suggestion
  further activity occurs. Thank you for your contribution.
  
  Please feel free to re-open and address the feedback and we'll take another look.
```

**Review by rinatkhaziev — CHANGES_REQUESTED** [review]
Looks good! I left a couple of minor wording suggestions, plus let's be less stringent for now (warn after 90 days and close after 14) - we can always bump it down.

**Comment by pschoffer** [comment]
Thank you @pauarge !

**Review by pschoffer — APPROVED** [review]

**Pull request merged**